meta-xt-driver-domain/recipes-extended/displbe/files/displbe.service

```diff
@@ -1,10 +1,11 @@
 [Unit]
 Description=Display backend
 Wants=xenstored.service
-After=xenstored.service weston@%u.service
+After=xenstored.service weston.service
 
 [Service]
 Type=simple
+Environment=WAYLAND_DISPLAY=wayland-1
 EnvironmentFile=/etc/default/weston
```

Comment:
It is unclear why "EnvironmentFile" didn't worked in your case. Can you please extend the commit message with the clarification?

Reply:
I was trying to put "WAYLAND_DISPLAY=wayland-1" into file pointed out by EnvironmentFile, but it didn't work as I expected it to work, even Weston didn't start. I haven't figured out why.

Reply:
"+"